install.sh

```diff
@@ -1,60 +1,51 @@
 #!/bin/bash
 
-# exit on fail
+# Exit on fail
 set -e
 
-#chromebrew directories
+# Default chromebrew repo values
+
 : "${OWNER:=chromebrew}"
 : "${REPO:=chromebrew}"
 : "${BRANCH:=master}"
+
 URL="https://raw.githubusercontent.com/${OWNER}/${REPO}/${BRANCH}"
+
+# Chromebrew directories
 : "${CREW_PREFIX:=/usr/local}"
 CREW_LIB_PATH="${CREW_PREFIX}/lib/crew"
 CREW_CONFIG_PATH="${CREW_PREFIX}/etc/crew"
 CREW_BREW_DIR="${CREW_PREFIX}/tmp/crew"
 CREW_DEST_DIR="${CREW_BREW_DIR}/dest"
-CREW_PACKAGES_PATH="${CREW_LIB_PATH}/packages"
 : "${CURL:=/usr/bin/curl}"
 : "${CREW_CACHE_DIR:=$CREW_PREFIX/tmp/packages}"
+
+# Architecture
+
 # For container usage, where we want to specify i686 arch
 # on a x86_64 host by setting ARCH=i686.
 : "${ARCH:=$(uname -m)}"
 # For container usage, when we are emulating armv7l via linux32
 # uname -m reports armv8l.
 ARCH="${ARCH/armv8l/armv7l}"
 
-# BOOTSTRAP_PACKAGES cannot depend on crew_profile_base for their core operations (completion scripts are fine)
-BOOTSTRAP_PACKAGES="musl_zstd pixz ca_certificates git gmp ncurses xxhash lz4 popt libyaml openssl zstd gcc rsync ruby"
+# Simplify colors and print errors to stderr (2)
+echo_error() { echo -e "\e[1;91m${*}${RESET}" >&2; } # Use Light Red for errors.
```

Comment:
I think you want `>&1` here too.  Otherwise, the error won't display.

Reply:
Actually, it's probably better to exclude `>&1` altogether.

Reply:
The `>&2` here means `> /dev/fd/2` (redirect stdout to stderr)

Reply:
A redirect to stderr will not print the result to stdout.

Reply:
That is expected...

Reply:
I haven't changed the original behavior here, so what is the conclusion? Do I use `>&1`?

Reply:
Leave it.  If we need to print to stdout we can add 2>&1 to the command.  FWIW, this is needed for several help commands currently, unfortunately.